### PR TITLE
Fix: add icons

### PR DESCRIPTION
**Pull request by Vikingjonsson**
Added new icons and renames alert to bell

### CODE REVIEW

File: src/Atoms/Icon/index.ts [hunk 1]
```diff
@@ -1,15 +1,18 @@
-import { Alert } from './components/Alert';
 import { ArrowRight } from './components/ArrowRight';
 import { Attachement } from './components/Attachement';
 import { Bank } from './components/Bank';
+import { Bell } from './components/Bell';
 import { Bolt } from './components/Bolt';
+import { DiagonalWithCircle } from './components/DiagonalWithCircle';
```
Comment (mjmonline) [inline src/Atoms/Icon/index.ts]: alphabetical order please

File: src/Atoms/Icon/index.ts [hunk 2]
```diff
@@ -30,18 +33,21 @@ import { WarningTriangle } from './components/WarningTriangle';
 import { Rtmt } from './components/Rtmt';
 
 export default {
-  Alert,
```
Comment (jauniuse) [inline src/Atoms/Icon/index.ts]: nah, this is breaking change. You should make it backwards compatible

Comment (jauniuse) [reply src/Atoms/Icon/index.ts]: if rename was really needed here, then export `Alert` also:

```js
export default {
  Alert: RenamedIcon,
  RenamedIcon,
}
```
and write a deprecation comment

Comment (mjmonline) [reply src/Atoms/Icon/index.ts]: he just put this in 5mins ago

Comment (jauniuse) [reply src/Atoms/Icon/index.ts]: sneaky

**Pull request merged**